recipe/meta.yaml

```diff
@@ -1,171 +1,156 @@
-    #- patches/0008-Add-alternate-clock_gettime-implementation-for-macOS.patch   # [osx]
-    - patches/unbundle_llvm.patch                                               # [linux]
+    - patches/0001-relax-tolerence-in-special_math_test.patch
+    - patches/0002-skip-read-only-debugger_cli_common-test.patch
+    - patches/0003-relax-precision-in-matrix_log_op_test.patch
+    - patches/0004-Unbundle-LLVM.patch
+    - patches/0005-rerun-generate_llvm_build.py.patch
 
 build:
   number: 0
   skip: True  # [not x86_64]
-  skip: True  # [win and vc<14]
 
 requirements:
   build:
-    - {{ compiler('c') }}       # [linux]
-    - {{ compiler('cxx') }}     # [linux]
-    - bazel >=0.24.1,<=0.25.2   # [linux]
-    - nasm                      # [linux]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    # see https://github.com/tensorflow/tensorflow/blob/v{{ version }}/.bazelversion
+    - bazel ==3.1.0
+    - nasm
   host:
     # conda build requirements
     - python
     - pip
-    - setuptools <=39.1.0
-    - swig                      # [linux]
-    # requirements specified by the package itself
-    - absl-py >=0.7.0
-    - astor >=0.6.0
-    - gast >=0.2.0
-    - google-pasta >=0.1.6
-    - keras-applications >=1.0.6
-    - keras-preprocessing >=1.0.5
-    - llvmdev >=9.0.0,<10.0.0    # [linux]
-    - zlib                      # [linux]
-    - libpng                    # [linux]
-    - libcurl                   # [linux]
-    - nsync                     # [linux]
-    - numpy 1.16.1
-    - six >=1.10.0
-    - protobuf >=3.6.1
+    # TODO: potentially remove these?
+    - llvmdev >=9.0.0
+    - zlib
+    - libpng
+    - libcurl
+    - curl         # [win]
+    - unzip        # [not win]
+    - zip          # [not win]
+    - m2-unzip     # [win]
+    - m2-zip       # [win]
+    - openjdk >=8
+    - nsync
+    # requirements specified by the package itself, see
+    # github.com/tensorflow/tensorflow/blob/v{{ version }}/tensorflow/tools/pip_package/setup.py
+    - absl-py >=0.10.0
+    - astunparse >=1.6.3
+    - flatbuffers >=1.12.0
+    - google-pasta >=0.2
+    - h5py >=2.10
+    - keras-preprocessing >=1.1.2
+    - numpy >=1.19.2
+    - opt_einsum >=3.3.0
+    - protobuf >=3.9.2
+    - six >=1.15.0
     - termcolor >=1.1.0
+    - typing_extensions >=3.7.4
+    - wheel >=0.35
     - wrapt >=1.11.1
-    - grpcio >=1.8.6
-    - wheel >=0.26              # [linux]
-    - mock >=2.0.0  # [py==27]
-    - backports.weakref >=1.0rc1  # [py==27]
-    - enum34 >=1.1.6  # [py==27]
-    - tensorflow-estimator >=1.14.0rc0,<1.15.0a0
+    # upstream wants exact pin for gast
+    - gast ==0.3.3
+    # TF-API needs to move in sync
+    - tensorboard >=2.4.0,<2.5
+    - tensorflow-estimator >=2.4.0,<2.5
   run:
     - {{ pin_subpackage('tensorflow-base', exact=True) }}
 
 outputs:
   - name: tensorflow-base
-    script: build_pkg.sh  # [linux]
-    script: build_pkg_osx.sh  # [osx]
-    script: build_pkg.bat  # [win]
```

Comment:
Since these scripts are removed, something like `files` should be used. See https://conda.io/projects/conda-build/en/latest/resources/define-metadata.html#specifying-files-to-include-in-output